packages/api-explorer/__tests__/JsonForm/__snapshots__/index.test.jsx.snap

```diff
@@ -44,14 +44,13 @@ exports[`JSONForm  snapshot 1`] = `
     type="submit"
   />
   <div
-    class="mia-container-wrapper"
+    class="je-object__container mia-container-wrapper"
```

Comment:
Why these changes?

Reply:
there shouldn't be these changes in fact. nothing was touched

Reply:
I try to understand as soon as I can